pkg-kni/pfpstatus/pfpstatus.go

```diff
@@ -18,26 +18,23 @@ package pfpstatus
 
 import (
 	"context"
-	"encoding/json"
 	"os"
-	"path/filepath"
-	"strings"
 	"time"
 
 	"github.com/go-logr/logr"
 
 	"github.com/k8stopologyawareschedwg/podfingerprint"
+	"github.com/openshift-kni/debug-tools/pkg/pfpstatus/record"
```

Comment:
why don't we replace all this package with `debug-tools/pfpstatus`? the new package is an extension of this one `pkg-kni/pfpstatus`.

Reply:
I don't see direct `debug-tools/pfpstatus` https://github.com/openshift-kni/debug-tools, am I missing something?